features/typed-arrays.yml.dist

```diff
@@ -0,0 +1,163 @@
+# Generated from: typed-arrays.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2015-07-29
```

Comment:
This is unchanged and matches https://caniuse.com/typedarrays except for Safari, where caniuse says that Safari 5.1 didn't support `Float64Array`. I haven't checked if that's right.